Projects/UOContent/Mobiles/Monsters/ML/Special/Ilhenir.cs

```diff
@@ -312,12 +296,8 @@ public StainedOoze(bool corrosive = false) : base(0x122A)
             m_Ticks = 0;
         }
 
-        public StainedOoze(Serial serial)
-            : base(serial)
-        {
-        }
-
         [CommandProperty(AccessLevel.GameMaster)]
+        [SerializableProperty(0)]
```

Comment:
Change this to a field _corrosive, instead.

Reply:
![image](https://user-images.githubusercontent.com/30377429/208167745-ed2d2195-3969-4c2b-b967-4ac64b5a702f.png)

Reply:
Do not need the property.